packages/react-instantsearch/src/widgets/SearchBox.tsx

```diff
@@ -28,12 +28,19 @@ export type SearchBoxProps = Omit<
      * @default true
      */
     searchAsYouType?: boolean;
+    /**
+     * Whether to update the search state in the middle of a
+     * composition session.
+     * @default false
+     */
+    ignoreCompositionEvents?: boolean;
     translations?: Partial<UiProps['translations']>;
```

Comment:
This describes what happens when this is `false` right ? I think this should be more clear as I got confused